api/analyzers/source_analyzer.py

```
@@ -81,61 +116,61 @@ def second_pass(self, ignore: List[str], executor: concurrent.futures.Executor)
-
-                task = executor.submit(process_file, file_path)
-                tasks.append(task)
-
-        # Wait for all tasks to complete
-        concurrent.futures.wait(tasks)
-
-    def analyze_file(self, path: Path, graph: Graph) -> None:
-        ext = path.suffix
-        logging.info(f"analyze_file: path: {path}")
+                for _, entity in file.entities.items():
+                    entity.resolved_symbol(lambda key, symbol: analyzers[file_path.suffix].resolve_symbol(self.files, lsps[file_path.suffix], file_path, key, symbol))
```

Ruff (0.8.2)

Function definition does not bind loop variable `file_path`

(B023)